assignments/objects.js

```diff
@@ -84,12 +84,16 @@ console.log(intern5.gender)
 // Give Kennan the ability to say "Hello, my name is Kennan!" Use the console.log provided as a hint.
 // console.log(kennan.speak());
 
-console.log("Hello my name is", intern2.name)
+intern2.speak= function (){
+  console.log("Hello my name is", this.name)
+}
+console.log(intern2.speak());
+
 
 // Antonietta loves math, give her the ability to multiply two numbers together and return the product. Use the console.log provided as a hint.
-//console.log(antonietta.multiplyNums(3,4));
+console.log(intern5.multiplyNums(3,4));
```

Comment:
I get `ReferenceError: multiplyNums is not defined` once I run ` console.log(intern5.multiplyNums(3,4));`. Can you find the error in your code, that causes this behaviour?